### PR TITLE
Correct typos in haskell-mode.texi

### PR DESCRIPTION
**Pull request by felixonmars**

### CODE REVIEW

File: doc/haskell-mode.texi [hunk 1]
```diff
@@ -63,7 +63,7 @@ interpreter (e.g. GHCi).
 * Unicode support::                      How to use Unicode
 * Indentation::                          Notes about indentation
 * External indentation::                 Other ways to indent code
-* Autoformating::                        Using external formatters
+* Autoformatting::                        Using external formatters
```
Comment (purcell) [inline doc/haskell-mode.texi]: Thanks - would you mind fixing the alignment here please?

**Comment by purcell** [comment]
Closing because submitted by a bot.

**Comment by felixonmars** [comment]
Hi, I'm not a bot :(
Will fix tomorrow.

**Comment by purcell** [comment]
Haha, okay... that's exactly what a really good bot would say though. 😂

**Comment by felixonmars** [comment]
@purcell Lol indeed... Fixed now :P

**Comment by purcell** [comment]
Thanks, much appreciated.

**Pull request merged**